R/msaGaugeRR.R

```diff
@@ -1011,7 +1015,7 @@ msaGaugeRR <- function(jaspResults, dataset, options, ...) {
     p1 <- p1 + ggplot2::scale_x_continuous(name = gettext(Xlab.StudySD))
 
 
-  if (ToleranceVar != ""){
+  if (ToleranceUsed == TRUE){
```

Comment:
please remove `== TRUE` everywhere.
```suggestion
  if (ToleranceUsed){
```